app/components-react/windows/go-live/dual-output/DualOutputGoLive.m.less

```diff
@@ -0,0 +1,183 @@
+@import '../../../../styles/index.less';
+@import '../GoLive.m.less';
+
+.settings-row {
+  height: 100%;
+  margin: 0px !important;
+
+  :global(.ant-col) {
+    padding: 0px;
+  }
+
+  .left-column {
+    height: 100%;
+    padding: 0px !important;
+
+    &:global(.ant-col) {
+      padding: 10px;
+    }
+  }
+
+  .right-column {
+    height: 100%;
+    background-color: var(--background);
+    overflow: hidden;
+
+    &:global(.ant-col.ant-col-16) {
+      padding: 25px !important;
+    }
+  }
+}
+
+.platform-switcher {
+  .radius();
+
+  display: flex;
+  margin: 15px;
+  padding: 10px;
+  flex-direction: column;
+  color: var(--title);
+  background: var(--dropdown-bg);
+  transition: transform 0.05s;
+  cursor: pointer;
+
+  .platform-disabled {
+    opacity: 0.7;
+    background: rgba(#2b383f, 0.3);
+  }
+
+  .switcher-header {
+    display: flex;
+    flex: 1;
+    justify-content: space-between;
+  }
+
+  .platform-info-wrapper {
+    white-space: nowrap;
+    display: flex;
+    flex: 1;
+    text-overflow: ellipsis;
+    overflow: visible;
+
+    i:not(.close) {
+      font-size: 35px;
+      margin: 2px;
+    }
+
+    .platform-logo {
+      .margin();
+
+      &-twitch {
+        font-size: 40px;
+      }
+      &-youtube {
+        font-size: 35px;
+        margin-top: 5px;
+      }
+      &-facebook {
+        font-size: 40px;
+      }
+      &-tiktok {
+        font-size: 40px;
+        margin-left: 5px;
+      }
+    }
+
+    .platform-info {
+      display: flex;
+      flex-direction: column;
+      flex-grow: 1;
+      margin: 0px 5px;
+
+      .platform-name {
+        font-size: 16px;
+        font-weight: 600;
+      }
+
+      .platform-username {
+        color: var(--link);
+        overflow-wrap: break-word;
+        display: inline-block;
+        word-break: break-word;
+        white-space: pre-line;
```

Comment:
usually we want to avoid highly nested less classes like this, especially if there aren't any namespace collisions which shouldn't happen with modules